File: Shell Commands/files/생일자역설검증.js
function birthdayParadox(population, numOfcase) {
    let remainingCase = numOfcase;
    let count = 0;
    let probability = 1;

    for (let i = 0; i < population; i++) {
        let probThis = 0;
        remainingCase = numOfcase - count++;
        probThis = remainingCase / numOfcase;
        probability *= probThis;
    }

    console.log(count, probability);

    return {
        probability,
        count,
    }
}

const {probability, count} = birthdayParadox(100000, 1073741824);

console.log(1 - probability);

File: Shell Commands/files/생일자역설검증.py
def birthdayParadox(population, numOfcase):
    remainingCase = numOfcase;
    count = 0;
    probability = 1;

    for i in range(population):
        probThis = 0
        remainingCase = numOfcase - i
        probThis = remainingCase / numOfcase;
        probability *= probThis
        count += 1

    return probability

prob = birthdayParadox(100000000, 2 ** 64)
print(1 - prob)